FrontEnd/HomeClubUI/app/Components/MobileNumberForm.js

```diff
@@ -10,23 +11,62 @@ import {
 } from 'react-native';
 
 export default class MobileNumberForm extends Component {
```

Comment:
React.Component

Reply:
Sorry i did not get your comment . please elaborate.